src/Model/Stercket.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Model;
+
+class Stercket
+{
+    private int $id;
+    private string $name;
+    private string $specie;
+    private string $type;
+    private int $attack;
+    private int $defense;
+    private int $health = 30;
+    private string $owner;
+
+    public function __construct(string $name, string $specie, string $type, string $owner)
+    {
+        $this->name = $name;
+        $this->specie = $specie;
+        $this->type = $type;
+        $this->owner = $owner;
+        $this->attack = $this->nrand(5, 0.5);
+        $this->defense = $this->nrand(2, 0.5);
+    }
```

Comment:
You probably should verify the input values to ensure a valid Stercket. For example, specie, type & owner should be one of a small amount of possible values
Maybe in the setters and use the setters here